__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -8556,6 +8556,1701 @@ Object {
 }
 `;
 
+exports[`#validate Postman support should support converting a Postman collection to OpenAPI (validating it in the process) 1`] = `
+Object {
+  "components": Object {
+    "securitySchemes": Object {
+      "apikeyAuth": Object {
+        "scheme": "apikey",
+        "type": "http",
+      },
+      "oauth2Auth": Object {
+        "scheme": "oauth2",
+        "type": "http",
+      },
+    },
+  },
+  "info": Object {
+    "description": "This is a sample server Petstore server.  You can find out more about Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).  For this sample, you can use the api key \`special-key\` to test the authorization filters.
+
+Contact Support:
+ Email: apiteam@swagger.io",
+    "title": "Swagger Petstore",
+    "version": "1.0.0",
+  },
+  "openapi": "3.0.0",
```

Comment:
This is what a converted collection looks like from `petstore-to-openapi`. It validates through our [@readme/openapi-parser](https://npm/@readme/openapi-parser) package!